### PR TITLE
feat: adding adaptSecurityContxt option to OpenShift alpha docs

### PR DESCRIPTION
**Pull request by hamza-m-masood**
## Description

My change relates to this issue: https://github.com/camunda/distribution/issues/294
I have added an option for the customer to set the value of `global.compatibility.openshift.adaptSecurityContext` to `false`
They do not require our OpenShift values.yaml anymore.
## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/self-managed/setup/deploy/openshift/redhat-openshift.md [hunk 1]
```diff
@@ -41,31 +41,27 @@ Depending on your OpenShift cluster's Security Context Constraints (SCCs) config
...
 # Make sure to set CHART_VERSION to match the chart version you want to install.
-helm pull camunda/camunda-platform --version "$CHART_VERSION" --untar --untardir "/tmp/camunda-platform-$CHART_VERSION"
-```
-
-5. Install the Camunda chart with the patched SCCs (`/tmp/camunda-platform-CHART_VERSION/camunda-platform/openshift/values.yaml`)
-
-```shell
 helm install camunda camunda/camunda-platform --skip-crds --version "$CHART_VERSION" \
-    --values "/tmp/camunda-platform-$CHART_VERSION/camunda-platform/openshift/values.yaml"
+  --set global.compatibility.openshift.adaptSecurityContext=disabled
```
Comment (conceptualshark) [inline docs/self-managed/setup/deploy/openshift/redhat-openshift.md:62]: Should this be `adaptSecurityContext=force` as in the instructions?

**Review by conceptualshark — DISMISSED** [review]

File: docs/self-managed/setup/deploy/openshift/redhat-openshift.md [hunk 2]
```diff
@@ -41,31 +41,27 @@ Depending on your OpenShift cluster's Security Context Constraints (SCCs) config
...
 To deploy Camunda 8 on OpenShift, please follow these installation steps:
 
 1. Install [Helm and other CLI tools](/self-managed/setup/install.md#prerequisites).
 2. Install the [Camunda Helm chart repository](/self-managed/setup/install.md#helm-repository).
-3. Download the exact version of the chart that you want to install and extract it in a directory ([Camunda 8 Helm Chart Version Matrix](https://helm.camunda.io/camunda-platform/version-matrix/)):
+3. Set `global.compatibility.openshift.adaptSecurityContext` to `force`
 
 ```shell
 # List of available versions: https://helm.camunda.io/camunda-platform/version-matrix/
 export CHART_VERSION="<DESIRED_CHART_VERSION>"
```
Comment (aabouzaid) [inline docs/self-managed/setup/deploy/openshift/redhat-openshift.md:58]: No need for this var anymore.

**Review by aabouzaid — CHANGES_REQUESTED** [review]

File: docs/self-managed/setup/deploy/openshift/redhat-openshift.md (hunk 2 above)
Comment (hamza-m-masood) [reply docs/self-managed/setup/deploy/openshift/redhat-openshift.md]: Removed

**Review by aabouzaid — DISMISSED** [review]

**Comment by hamza-m-masood** [comment]
@conceptualshark Feel free to merge.

**Review by conceptualshark — DISMISSED** [review]

**Review by conceptualshark — APPROVED** [review]

**Pull request merged**